### ISSUE TITLE
Stored procedures

### ISSUE THREAD

**Issue by tokenrove** [propublica/nyc-dna-software#4]
To validate many of the routines, it's necessary to validate the stored procedures they call, but the database schema is presumably only present in the MSSQL `.bak` blobs; could someone with a copy of MSSQL dump those to plain text SQL?

**Comment by d3ullist** [comment]
Dumps made with [rebase data](https://www.rebasedata.com)

[FST_DB.zip](https://github.com/propublica/nyc-dna-software/files/1434743/FST_DB.zip)
[FST_Membership.zip](https://github.com/propublica/nyc-dna-software/files/1434746/FST_Membership.zip)

**Comment by tokenrove** [comment]
Thanks!  But the stored procedures do not appear to be here, only the data.  I think we'll want a full dump that includes `information_schema` if it exists; if it doesn't exist in the BAKs at all, then the source has not fully been released. :-/

**Comment by sbronson** [comment]
[fst.zip](https://github.com/propublica/nyc-dna-software/files/1444504/fst.zip)

**Comment by tokenrove** [comment]
There we go.  Very nice!  Scary cargo cult use of `SELECT TOP (100) PERCENT` which doesn't even work consistently between versions, AFAIK.  We should probably commit that into the repo.

**Comment by drchriscole** [comment]
Yeah, I've been wondering about that. Will ProPublica accept pull requests? What do you think @laurenkirchner ?

Probably best to keep trunk clean and create a new dev branch.
